fab/environments.yml

```diff
@@ -477,3 +477,67 @@ swiss:
         num_processes: 1
       XFormToElasticsearchPillow:
         num_processes: 1
+
+l10k:
+  environment: l10k
+  jython_memory: "1024m"
+  formplayer_memory: "1024m"
+
+  service_blacklist:
+    - supervisor_websockets.conf
+
+  celery_processes:
+    '*':
+      main:
+        concurrency: 1
+      background_queue:
+        concurrency: 1
+      email_queue:
+        concurrency: 1
+      saved_exports_queue:
+        concurrency: 1
+      pillow_retry_queue:
+        concurrency: 1
+      ucr_queue:
+        concurrency: 1
+      async_restore_queue:
```

Comment:
wonder if we can get away with only a couple queues. we probably don't need this, but suppose it doesn't make too much of a difference now that we use autoscale